src/main/resources/l10n/JabRef_da.properties

```diff
@@ -1460,7 +1460,8 @@ Looking_for_full_text_document...=Søger_efter_tekstdokument...
 Autosave=Automatisk_sikkerhedskopi
 A_local_copy_will_be_opened.=
 Autosave_local_databases=
-Autosave_shared_databases=
+Autosave_the_database_to=
+Please_enter_a_valid_file_path.=
```

Comment:
Isn't there another string available when the user uses the normal save as or is this functionality missing at the other save operation, too?